### ISSUE TITLE
Reference material

### ISSUE THREAD

**Issue by Sealjay** [Green-Software-Foundation/software_carbon_intensity#20]
@atg-abhishek talked about having a place to add additional reference and background material, like the background on fuel mix that @Henry-WattTime shared today.

That could be good, at least as reference material or external sources for new joiners to the GSF.

Do we have an appropriate place outside of the specification to reference this @seanmcilroy29  ? Maybe as an addendum to the dictionary, or as background to the way we work? Slack is great for ongoing discussions, but it might be worth being able to link to sources or material that informed our viewpoint.

**Comment by jawache** [comment]
There are two places on the spec for some types of references already, see: https://github.com/Green-Software-Foundation/software_carbon_intensity/blob/dev/Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md#references

- Normative References - are like pre-reading, the things you MUST read and understand before you can understand the topics in this spec.
- Informative References - are just generally related content to this content, you don't have to read them, but it can be useful.

**Comment by Sealjay** [comment]
I'd seen those, but they seemed a little sparse compared to what we saw on the call! I guess we just need to do the work of adding the information

**Comment by jawache** [comment]
Yeah, what's there is from that template doc we cloned from. We haven't
added or updated anything in that section yet, I think if someone does the
first PR others will follow 😉

**Comment by atg-abhishek** [comment]
Actually @Sealjay, I was discussing this with @seanmcilroy29 and suggested that we create a "Welcome / Beginner's Guide to Green Computing" that _may_ sit outside as an `FAQ document` so that folks can quickly get caught up with major concepts that relate to sustainable computing. Thoughts @jawache ?
